src/Components/Navbar.jsx

```diff
@@ -0,0 +1,108 @@
+import React, { useState } from "react";
+import "./Navbar.css";
+import {RiMenu3Line, RiCloseLine} from "react-icons/ri";
+import logo from "../assets/logo.jpg";
+
+const Navbar=() => {
+    const [toggleMenu, setToggleMenu] = useState(false)
```

Comment:
**suggestion:** Ich würde boolean Variablennamen immer mit `is`, `has`, `can` oder Ähnlichem beginnen: `isActive`, `hasContent`, `canSubmit`,...

Dadurch ist auf den ersten Blick klar, um was für einen Datentypen es sich handelt und in welchem Fall die Variable `true` oder `false` ist.

In diesem Fall also z.B. was wie `isMenuOpen`

Reply:
Suggestion übernommen